slsim/Sources/source_variability/variability.py

```diff
@@ -42,3 +43,29 @@ def sinusoidal_variability(t, amp, freq):
     :return: variability for the given time
     """
     return amp * np.sin(2 * np.pi * freq * t)
+
+
+def interpolate_variability(Image_series, Orig_timestamps, New_timestamps):
+    """Interpolates a variable source to any given time series.
+
+    :param Image_series: 3 dimensional array of shape (t, x, y) relating to snapshots of
+        a variable object in source plane
+    :param Orig_timestamps: Series of timestamps which represent original snapshots
```

Comment:
It's unclear to me how this input differs from the first axis t in the Image_series. In general, having an explicit definition of these inputs and describing what is happening would be useful. It's unclear to me when reading the docstrings what this function does

Reply:
Made more clear